### DISCUSSION TITLE
Ray does not work on PHP 8.0.7

### DISCUSSION THREAD

**Discussion by mxrxdxn** [spatie/ray#493] [Q&A]
**Describe the bug**
Running under PHP 8.0.7, any calls to the `ray()` function results in a browser error "page returned no data".

**Versions**
Ray version (you can see this in "About Ray"):

You can use `composer show` to get the version numbers of:
- spatie/ray package version: 1.22.1 (also tested on latest version)
- spatie/laravel-ray package version (if applicable): 1.17.2 (also tested on latest version)

PHP version: 8.0.7
Laravel version (if applicable): 8.40

**To Reproduce**
1. Add a ray call to your codebase (`ray('test')` will trigger the issue).
2. Execute.

**Expected behavior**
The data passed to `ray()` is sent to the Ray desktop app.

**Screenshots**
![Ray error](https://p28.p0.n0.cdn.getcloudapp.com/items/2NuqmykQ/f856b9ea-371a-40ce-afe4-11657d4d4500.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Version Big Sur (11.4)
 
 **Additional context**
My environment was running on 8.0.6 and Ray was working fine on that version of PHP - it has only stopped working since upgrading to 8.0.7.
I have also tested this in a Laravel project and in a WordPress project, and in both instances page output is killed.

**Comment by freekmurze** [comment]
I can't seem to reproduce this, Could you create an example app in which ray is not working for you?

**Comment by mxrxdxn** [comment]
Yeah - I can confirm this isn't related to PHP 8.0.7. I've installed PHP 7.4 and 7.3 (which previously worked with Ray too) and both of those now also display the same behaviour as above.

I'm really not sure what has caused this; my Homebrew performed an update on Friday which is likely the culprit, however I'm now unsure as to what has been changed which has caused this issue.

**Comment by mxrxdxn** [comment]
After some more investigation I believe this to be an issue between this package and PHP IMAP provided by the shivammathur/homebrew-extensions homebrew tap.

**Reply by freekmurze** [reply]
If there's something we can do from our end to fix this, let us know!
